packages/synthchron-simulator/src/simulation.ts

```diff
@@ -35,16 +36,17 @@ const getSimulationProgress = <SpecificProcessModel, StateType>(
   processEngine: ProcessEngine<SpecificProcessModel, StateType>,
   processModel: SpecificProcessModel,
   simulationConfiguration: Configuration,
-  simulationResults: TraceSimulationResult[]
+  simulationResults: TraceSimulationResult[],
+  steps: number
```

Comment:
Isn't the size of the trace = number of steps? or it's to make the function more general

Reply:
No, since due to unique traces we can have less results than steps. Imagine generating the same trace 10 times. You'd only have a result length of 1, but 10 steps